client/layout/masterbar/drafts.jsx

```diff
@@ -79,8 +80,21 @@ class MasterbarDrafts extends Component {
 				onTouchStart={ this.preload }
 				onMouseEnter={ this.preload }
 				ref={ this.setDraftsRef }
+				aria-haspopup
+				aria-expanded={ this.state.showDrafts || undefined }
 			>
-				<Count count={ this.props.draftCount } />
+				<span className="masterbar__toggle-drafts-lg-label">
+					{ this.props.numberFormat( this.props.draftCount ) }
+				</span>
+
+				{ /* Don't display draft count on small screens when the count has more than 2 digits */ }
+				{ this.props.draftCount < 100 && (
```

Comment:
Maybe 100 is fine, But what about 100,000?
Obviously that's a pretty broken situation to be in, but that's why I figured it wasn't worth doing the styling for it. Dealing with 2 digits seems like enough to me.

Originally I was just going to hide the draft count at the same break point that we hide the "Write" label. Ollie suggested we keep it if there's up to 2 digits because he didn't want to regress it in case someone was depending on seeing the number of drafts.

Reply:
Or maybe we could show `99+` on small screens. But I think it's also good to hide 👍